docs/user/cli/geotools.rst

```diff
@@ -0,0 +1,102 @@
+
+Setting up the Command Line Tools
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once installed, the command line tools can be invoked by running the script ``geomesa-gt`` located in the binary
+distribution under ``geomesa-gt_2.11-$VERSION/bin/``.
+
+The tools ship with some default GeoTools data stores, such as support for Postgis and shapefiles. For other stores,
+you will need to copy the appropriate JARs into the tools ``lib`` folder.
+
+.. note::
+
+    You can configure environment variables and classpath settings in ``geomesa-gt_2.11-$VERSION/conf/geomesa-env.sh``.
+
+.. note::
+
+    ``geomesa-gt`` will load JARs from the ``$GEOMESA_EXTRA_CLASSPATHS`` environment variable
+    into the class path. Use the ``geomesa-gt classpath`` command in order to see what JARs are being used.
+
+
+Due to licensing restrictions, dependencies for shape file support must be separately installed. Do this with the
+following commands:
+
+.. code-block:: bash
+
+    $ bin/install-jai.sh
+    $ bin/install-jline.sh
+
+Run ``geomesa-gt`` without arguments to confirm that the tools work.
+
+.. code::
+
+    $ bin/geomesa-gt
+    INFO  Usage: geomesa-gt [command] [command options]
+      Commands:
+      ...
+
+Setting up Distributed Processing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+GeoMesa supports running map/reduce jobs for ingest and export. If you have a local Hadoop installation,
+the tools will read the ``$HADOOP_HOME`` environment variable to load the appropriate JAR files for Hadoop.
+
+If you do not have a local Hadoop installation, then in order to run distributed jobs you will need to manually
+install the Hadoop configuration files into the tools ``conf`` folder, and the Hadoop JARs into the tools ``lib``
+folder. To install JARs, use the script provided with the distribution:
+
+.. code-block:: bash
+
+    $ bin/install-hadoop.sh lib
+
+If you installed JARs for any additional data stores, you will need to add them to the Hadoop libjars path
+by modifying the file ``org/locationtech/geomesa/geotools/tools/ingest-libjars.list`` inside the
+``lib/geomesa-gt-tools_2.11-$VERSION.jar`` JAR.
+
+General Arguments
+-----------------
+
+Most commands require you to specify the connection to your data store. Parameters can be passed in using the
+``--param`` argument, which can be repeated in order to specify multiple parameters. Alternatively, the ``--params``
+argument can be used to specify a Java properties file containing the parameters. This may be useful for
+simplifying the command invocation, or to hide sensitive parameters from bash history and process lists.
+
+For example, to connect to a Postgis data store, you may use the following command:
+
+.. code::
+
+    $ bin/geomesa-gt export --param dbtype=postgis --param host=localhost \
+      --param user=postgres --param passwd=postgres --param port=5432 \
+      --param database=example --feature-name gdelt
```

Comment:
Hmm...  with this --param option...  I wonder if this would be a good way to allow for extra datastore params to be passed into our other command line tools.

Reply:
we could, yeah - were there any params in particular you were thinking of? generally it's nicer to have named parameters so that they can be validated, enforced as required/not, etc.

Reply:
It is nicer.  I'm just wondering if this is useful as an escape hatch in case we miss one...  Eh, maybe not?